scripts/build/config.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-process-env */
+
+const args = process.argv.filter((value, index) => index >= 2);
+const useAnalyser = args.find((value) => value === '--analyser');
+const isProd = args.find((value) => value === '--production');
+
+const rootPath = `${__dirname}/../../`;
+const targetPath = isProd ? `${rootPath}lib/` : `${rootPath}examples/packages/@shopify/draggable`;
```

Comment:
in production it writes into the lib folder and for development it will place the bundle in the examples pages